bhx5chain/CHANGELOG.md

```diff
@@ -8,6 +8,19 @@ Versioning](https://doc.rust-lang.org/cargo/reference/semver.html).
 
 ## [Unreleased]
 
+## 0.2.0 - 2025-04-04
```

Comment:
You can just enclose this in `[0.2.0]` and add a footnote:
```
[0.2.0]: <https://github.com/blockhousetech/eudi-rust-core/releases/tag/bhx5chain/v0.2.0>
```

Even though this link is not valid yet.  It will be when we make a GitHub release.

NOTE: I'd rather we test the newly added release & publish GitHub action while we are both in the office.  Monday?